src/simulators/state_chunk.hpp

```diff
@@ -419,7 +420,8 @@ class StateChunk : public State<state_t> {
                                const Noise::NoiseModel &noise,
                                ExperimentResult &result,
                                uint_t rng_seed,
-                               bool final_ops);
+                               bool final_ops,
+                               bool pauli_only);
```

Comment:
can we rename `pauli_only` with `batched_pauli_ops`?

Reply:
removed